common/utils/src/main/resources/error/error-classes.json

```diff
@@ -1956,7 +1974,13 @@
       "Not allowed to implement multiple UDF interfaces, UDF class <className>."
     ]
   },
-  "NAMED_ARGUMENTS_SUPPORT_DISABLED" : {
+  "NAMED_PARAMETERS_NOT_SUPPORTED" : {
```

Comment:
nit: The object is named as `object NamedArgumentsSupport`. Could you unify the naming on the 'parameters' and 'arguments'?

Reply:
nvm, it seems reasonable for the naming of `NamedArgumentsSupport ` but `NAMED_PARAMETERS_NOT_SUPPORTED`.

Reply:
Serge was actually the one who recommended the name change. It should be fine.

Reply:
Right, I agree with the change in the error class to parameter, but I was thinking if similar changes should be applied on `NamedArgumentsSupport`, e.g. `NamedParametersSupport`.

Reply:
Okay, that makes sense.